compiler/noirc_frontend/src/hir/def_collector/errors.rs

```diff
@@ -228,6 +230,15 @@ impl<'a> From<&'a DefCollectorErrorKind> for Diagnostic {
             DefCollectorErrorKind::MacroError(macro_error) => {
                 Diagnostic::simple_error(macro_error.primary_message.clone(), macro_error.secondary_message.clone().unwrap_or_default(), macro_error.span.unwrap_or_default())
             },
+            DefCollectorErrorKind::UnsupportedNumericGenericType { ident, typ } => {
+                let name = &ident.0.contents;
+
+                Diagnostic::simple_error(
+                    format!("{name} has a type of {typ}. The only supported types of numeric generics are integers, fields, and booleans."),
```

Comment:
```suggestion
                    format!("{name} has a type of {typ}. The only supported types of numeric generics are integers and fields"),
```
We don't support booleans yet, right?

Reply:
Yes good catch